src/main/java/io/github/moulberry/notenoughupdates/core/util/StringUtils.java

```diff
@@ -85,7 +85,7 @@ public static String shortNumberFormat(BigInteger bigInteger) {
 	}
 
 	public static String shortNumberFormat(double n, int iteration) {
-		if (n < 1000) {
+		if (n < 1000 && n > -1000) {
```

Comment:
I feel like it would be cleaner to have a just call itself like `return "-" + shortNumberFormat(-n, iteration)`